resources/asciidoctor/spec/cramped_include_spec.rb

```diff
@@ -17,60 +16,40 @@
   end
 
   include_examples "doesn't break line numbers"
+  include_context 'convert without logs'
 
-  it "allows cramped includes of callout lists" do
-    actual = convert <<~ASCIIDOC
-      = Test
+  context 'when including callout lists without a blank line between them' do
+    let(:input) do
+      <<~ASCIIDOC
+        = Test
 
-      == Test
+        == Test
 
-      include::resources/cramped_include/colist1.adoc[]
-      include::resources/cramped_include/colist2.adoc[]
-    ASCIIDOC
-    expected = <<~DOCBOOK
-      <chapter id="_test">
-      <title>Test</title>
-      <section id="P1">
-      <title>P1</title>
-      <section id="P1_1">
-      <title>P1.1</title>
-      <programlisting language="java" linenumbering="unnumbered">words <1> <2></programlisting>
-      <calloutlist>
-      <callout arearefs="CO1-1">
-      <para>foo</para>
-      </callout>
-      </calloutlist>
-      </section>
-      </section>
-      <section id="P2">
-      <title>P2</title>
-      <section id="P2_1">
-      <title>P2.1</title>
-      <programlisting language="java" linenumbering="unnumbered">words <1> <2></programlisting>
-      <calloutlist>
-      <callout arearefs="CO2-1">
-      <para>foo</para>
-      </callout>
-      </calloutlist>
-      </section>
-      </section>
-      </chapter>
-    DOCBOOK
-    expect(actual).to eq(expected.strip)
+        include::resources/cramped_include/colist1.adoc[]
+        include::resources/cramped_include/colist2.adoc[]
+      ASCIIDOC
+    end
+    it 'renders both callout lists' do
+      expect(converted).to include('<callout arearefs="CO1-1">')
+        .and(include('<callout arearefs="CO2-1">'))
+    end
```

Comment:
Is there a reason you didn't write it like the following?

```ruby
expect(converted).to include('<callout arearefs="CO1-1">')
expect(converted).to include('<callout arearefs="CO2-1">')
```

Reply:
I suppose the error message is a *little* more descriptive the way I have it, but not much. Either way is fine by me. If you prefer your way I'll switch it.